CONTRIBUTING.md

```diff
@@ -67,6 +67,11 @@ lets us merge or address your contributions quickly.
    the issue tracker clean. The issue is still indexed and available for
    future viewers, or can be re-opened if necessary.
 
+## Pull Requests
+
+Pull requests must always be opened from a fork of `doctl`, even if you have
+commit rights to the repository.
+
```

Comment:
Would we want to add reasoning behind this? e.g. "To support tooling" or such

Reply:
It's to better support our open source collaborators. :)

Reply:
Added Andrew's suggestion 🙂